docs/software/python-cli/usage.mdx

```diff
@@ -29,19 +70,141 @@ INFO  | 18:38:04 711 [DeviceTelemetryModule] Sending packet to phone
+    network.wifi_enabled
+    network.wifi_psk
+    network.wifi_ssid
+display:
+    display.auto_screen_carousel_secs
+    display.compass_north_top
+    display.compass_orientation
+    display.displaymode
+    display.flip_screen
+    display.gps_format
+    display.heading_bold
+    display.oled
+    display.screen_on_secs
+    display.units
+    display.wake_on_tap_or_motion
+
+...
+```
+
+## Getting a list of User Preferences including the Values they are currently set to
 
-You can also use this tool to set any of the device parameters which are stored in persistent storage. For instance, here's how to set the device
-to keep the Bluetooth link alive for eight hours (any usage of the Bluetooth protocol from your phone will reset this timer)
+You can get this by running `--info` which reads and displays the radio config information. Which will also include a list of all "Nodes in mesh" known to the device. 
+
+```shell
+meshtastic --info
+```
+
+```shell title="A snippet of what you would see in the Resulting Output"
+# A snippet of what you would see in the Resulting Output:
+...
+
+ "power": {
+    "waitBluetoothSecs": 60,
+    "sdsSecs": 4294967295,
+    "lsSecs": 300,
+    "minWakeSecs": 10,
+    "isPowerSaving": false,
+    "onBatteryShutdownAfterSecs": 0,
+    "adcMultiplierOverride": 0.0,
+    "deviceBatteryInaAddress": 0,
+    "powermonEnables": "0"
+  },
+  "network": {
+    "ntpServer": "0.pool.ntp.org",
+    "wifiEnabled": false,
+    "wifiSsid": "",
+    "wifiPsk": "",
+    "ethEnabled": false,
+    "addressMode": "DHCP",
+    "rsyslogServer": ""
+  },
+  "display": {
+    "screenOnSecs": 1,
+    "compassNorthTop": true,
+    "headingBold": true,
+    "gpsFormat": "DEC",
+    "autoScreenCarouselSecs": 0,
+    "flipScreen": false,
+    "units": "METRIC",
+    "oled": "OLED_AUTO",
+    "displaymode": "DEFAULT",
+    "wakeOnTapOrMotion": false,
+    "compassOrientation": "DEGREES_0"
+  },
+
+...
+```
+
+## Changing Device Settings
 
+You can also use this tool to set any of the device parameters which are stored in persistent storage. 
+
+### How to set a specific preference
+
+Lets assume your device/node has a display and you want it to automatically cycle through the different screens.
+As we can see in the above "Resulting Output" snippet the `autoScreenCarouselSecs` (which is under `"display":`) is currently set to `0`. (Which means, "cycling" is currently disabled.)
+
+How to change that to, let´s say 20 seconds?
+We will use the `--set` attribute to change `autoScreenCarouselSecs`. But simply trying to use:
+
+```shell
+meshtastic --set autoScreenCarouselSecs 20
+```
+will result in the message:
+
+```shell title="Error Message"
+...
+LocalConfig and LocalModuleConfig do not have attribute autoScreenCarouselSecs.
+...
+```
+We will need to add the `display` part to our command by adding it before the `autoScreenCarouselSecs`. And we will use a `.` as seperator between them.
+So the command would be: 
```

Comment:
We don't need to showcase the wrong way to do it, we can just show the right way.